_inc/footer.php

```diff
@@ -28,7 +28,7 @@
 					<a href="http://jetpack.me/contact-support/" title="<?php esc_attr_e( 'Contact the Jetpack Happiness Squad.', 'jetpack' ); ?>"><?php _e( 'Support', 'jetpack' ); ?></a>
 					<a href="http://jetpack.me/survey/?rel=<?php echo JETPACK__VERSION; ?>" title="<?php esc_attr_e( 'Take a survey.  Tell us how we&#8217;re doing.', 'jetpack' ); ?>"><?php _e( 'Give Us Feedback', 'jetpack' ); ?></a>
 
-					<?php if ( $is_active && current_user_can( 'jetpack_disconnect' ) ) : ?>
+					<?php if ( $is_active && current_user_can( 'jetpack_disconnect' ) && ! Jetpack::is_development_mode() ) : ?>
```

Comment:
Wouldn't this be unnecessary if we modified $is_active so it's never showing as active in dev mode?